### PR TITLE
Update linting

### PR DESCRIPTION
**Pull request by kat1906**

### CODE REVIEW

File: app/sections/view-native-desktop/macro.test.js [hunk 1]
```diff
@@ -142,17 +142,17 @@ describe('view-native-desktop', () => {
         const $ = cheerio.load(res.text);
 
         const nativeDesktopSectionTable = $('[data-test-id="view-section-table-native-desktop"]');
-        const storageRequirementsDescriptionQuestionRow = nativeDesktopSectionTable.find('[data-test-id="view-section-table-row-storage-requirements-description"]');
-        const storageRequirementsDescriptionInnerComponent = storageRequirementsDescriptionQuestionRow
+        const storageReqsDescriptionQuestionRow = nativeDesktopSectionTable.find('[data-test-id="view-section-table-row-storage-requirements-description"]');
+        const storageReqsDescriptionInnerComponent = storageReqsDescriptionQuestionRow
```
Comment (kat1906) [inline app/sections/view-native-desktop/macro.test.js]: this line was too long so renamed the variables

File: app/sections/view-native-mobile/macro.test.js [hunk 2]
```diff
@@ -180,17 +180,17 @@ describe('view-native-mobile', () => {
         const $ = cheerio.load(res.text);
 
         const nativeMobileSectionTable = $('[data-test-id="view-section-table-native-mobile"]');
-        const storageRequirementsDescriptionQuestionRow = nativeMobileSectionTable.find('[data-test-id="view-section-table-row-storage-requirements-description"]');
-        const storageRequirementsDescriptionInnerComponent = storageRequirementsDescriptionQuestionRow
+        const storageReqsDescriptionQuestionRow = nativeMobileSectionTable.find('[data-test-id="view-section-table-row-storage-requirements-description"]');
+        const storageReqsDescriptionInnerComponent = storageReqsDescriptionQuestionRow
```
Comment (kat1906) [inline app/sections/view-native-mobile/macro.test.js]: this line was too long so renamed the variables

File: app/scripts/main.js [hunk 3]
```diff
@@ -1,13 +0,0 @@
...
-// import nhsuk_skipLink from '../../node_modules/nhsuk-frontend/packages/components/skip-link/skip-link';
-// import autocomplete from '../../node_modules/nhsuk-frontend/packages/components/header/headerAutoComplete';
-
-// // HTML5 polyfills
-// import '../../node_modules/nhsuk-frontend/packages/components/details/details';
-
-// // Initialise components
-// nhsuk_header();
-// nhsuk_skipLink();
-// autocomplete();
```
Comment (alibombaye) [inline app/scripts/main.js]: Can we delete this file?

Comment (kat1906) [reply app/scripts/main.js]: I don't know ... can we? I didn't want to delete it then somewhere complain about it not being found. I'll take a look and see

Comment (alibombaye) [reply app/scripts/main.js]: 🤷‍♂ I can't remember why we commented it out in the first place. But yeah let's see if we can just remove it

Comment (kat1906) [reply app/scripts/main.js]: leaving in as it makes webpack load correctly

**Pull request merged**